core/src/main/java/org/testcontainers/utility/TestcontainersConfiguration.java

```diff
@@ -71,13 +82,18 @@ public String getPulsarImage() {
     }
 
     public boolean isDisableChecks() {
-        return Boolean.parseBoolean((String) properties.getOrDefault("checks.disable", "false"));
+        return Boolean.parseBoolean((String) environmentProperties.getOrDefault("checks.disable", "false"));
     }
 
     public String getDockerClientStrategyClassName() {
-        return (String) properties.get("docker.client.strategy");
+        return (String) environmentProperties.get("docker.client.strategy");
```

Comment:
does not make sense to have it "for everyone" as well since it is strictly environment specific